docs/source/notebooks/regridding.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>@tlvu Is this change not dangerous?</p><p>I was unable to put both a cell magic ("%%") and the NBVAL ignore output coment...)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/pavics-sdi/pull/216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Indeed but I got around it using regex here https://github.com/Ouranosinc/PAVICS-e2e-workflow-tests/blob/d7d7a55e0aa83b2df2dc31bd3ae06219de5047e2/notebooks/output-sanitize.cfg#L102-L110, it's fine.</p><p> </p>